R/bundle.R

```diff
@@ -263,7 +288,9 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters, a
   metadata <- list(appmode = appMode)
 
   # emit appropriate primary document information
-  primaryDoc <- ifelse(is.null(appPrimaryDoc), NA, appPrimaryDoc)
+  primaryDoc <- ifelse(is.null(appPrimaryDoc) ||
```

Comment:
What's the status of this PR? (are we good to merge based on all of the
feedback/review)

On Mon, Oct 26, 2015 at 5:20 PM, Jonathan <notifications@github.com> wrote:

> In R/bundle.R
> <https://github.com/rstudio/rsconnect/pull/80#discussion_r43054887>:
>
> > @@ -263,7 +288,9 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters, a
> >    metadata <- list(appmode = appMode)
> >
> >    # emit appropriate primary document information
> > -  primaryDoc <- ifelse(is.null(appPrimaryDoc), NA, appPrimaryDoc)
> > +  primaryDoc <- ifelse(is.null(appPrimaryDoc) ||
>
> No, this code is temporary, and exists to maintain the facade for Connect
> that this is just a run-of-the mill Shiny app. In the future we'll always
> write the primaryDoc to the manifest, and Connect will use it to form the
> argument to runApp.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rstudio/rsconnect/pull/80/files#r43054887>.
>